.github/workflows/nerdctl.yaml

```diff
@@ -52,17 +50,19 @@ jobs:
 
       - name: Install nerdctl
         run: |
+          # Remove Docker and Podman
           sudo systemctl is-active --quiet docker.service || systemctl stop docker.service
-          sudo apt-get remove -y docker-ce docker-ce-cli docker-buildx-plugin podman
-          # Install Containerd
-          sudo curl -sSL https://github.com/containerd/containerd/releases/download/v${CONTAINERD_VERSION}/containerd-${CONTAINERD_VERSION}-linux-amd64.tar.gz|sudo tar -xvz -C /usr
-          sudo systemctl restart containerd.service
+          sudo apt-get remove -y docker-ce docker-ce-cli podman containerd.io
+          # Install nerdctl full package
+          sudo curl -sSL https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-full-${NERDCTL_VERSION}-linux-amd64.tar.gz | sudo tar -xvz -C /usr/local
+          # Start Containerd
+          sudo curl -sSL https://raw.githubusercontent.com/containerd/containerd/main/containerd.service > containerd.service
```

Comment:
I prefer to have pinned dependencies in ci and not a moving target

Reply:
Thanks @aojea 

It has been changed by using the `containerd.service` included by `nerd-full` pacakge.